docs/source-2.0/spec/authentication-traits.rst

```diff
@@ -372,3 +372,87 @@ authentication scheme trait that is not applied to the service:
 
 
 .. _IANA Authentication Scheme Registry: https://www.iana.org/assignments/http-authschemes
+
+
+.. _disabling-auth:
+
+Disabling Authentication
+========================
+
+Services should, in general, have authentication configured. However, there are
+times when authentication may need to be disabled for the service or for a
+number of operations within a service. For example, you may want to expose a
+public route for users to use to login that does not require authentication.
+
+
+Disabling Authentication for a Service
+--------------------------------------
+
+In Smithy, a service’s authentication schemes are determined by the presence
+of a trait with the :ref:`@authDefinition <authDefinition-trait>` meta trait.
+If you do not want your service to have authentication, do not apply any
+authentication traits to the service shape.
+
+The following example shows a service with no auth trait configured:
+
+.. code-block:: smithy
+
+    $version: "2.0"
+
+    namespace com.example
+
+    service MyService {
+        version: "2020-08-31",
+        operations: [
+            MyOperation,
+        ]
+    }
+
+    operation MyOperation {}
+
+
+Disabling Authentication for an Operation
```

Comment:
I'm not sure if all this content from guide needs to be here as-is. The [auth trait](https://smithy.io/2.0/spec/authentication-traits.html#auth-trait) section already has "The following example defines all combinations in which auth can be applied to services and operations:" which covers multiple cases of auth, just missing an explicit case for disabling auth for an operation. It seems better to just add a case to that example, instead of this separate section just for disable?

Similarly, disable for a service seems like it could be covered under the https://smithy.io/2.0/spec/authentication-traits.html#authdefinition-trait section, by adding an example of service with no authentication.